wiki/redirect/osu!academy/de.md

```diff
@@ -0,0 +1 @@
+/wiki/Announcements/osu!academy/?locale=de
```

Comment:
> Testing for osu-web!
For when I type `https://new.ppy.sh/help/wiki/osu!academy/?locale=de` instead of `https://new.ppy.sh/help/wiki/Announcements/osu!academy/?locale=de`